portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerTest.scala

```diff
@@ -0,0 +1,52 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.dimafeng.testcontainers.{ForAllTestContainer, PostgreSQLContainer}
+import com.typesafe.config.Config
+import cromwell.pipeline.ApplicationComponents
+import cromwell.pipeline.datastorage.dto.{Project, ProjectId, User, UserId}
+import cromwell.pipeline.utils.auth.{AccessTokenContent, TestContainersUtils, TestProjectUtils, TestUserUtils}
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
+import org.scalatest.{AsyncWordSpec, Matchers}
+
+class ProjectControllerTest
+  extends AsyncWordSpec
+    with Matchers
+    with ScalatestRouteTest
+    with PlayJsonSupport
+    with ForAllTestContainer {
+
+  override val container: PostgreSQLContainer = TestContainersUtils.getPostgreSQLContainer()
+  container.start()
+  implicit val config: Config = TestContainersUtils.getConfigForPgContainer(container)
+  private val components: ApplicationComponents = new ApplicationComponents()
+  override protected def beforeAll(): Unit = components.datastorageModule.pipelineDatabaseEngine.updateSchema()
+
+  import components.controllerModule.projectController
+  import components.datastorageModule.projectRepository
+  import components.controllerModule.userController
```

Comment:
Do you need this?

Reply:
Yis. Everything except components.controllerModule.userController

Reply:
Rename file

Reply:
Ok

Reply:
Done